script.js

```diff
@@ -0,0 +1,10 @@
+function sum() {
+    var a,b;
+        var a =Number(document.getElementById("a").value)
+        var b =Number(document.getElementById("b").value)
+     sum = a + b 
+        var answer = parseInt(a) + parseInt(b)
+        document.getElementById('answer').innerHTML = answer
```

Comment:
Be consistent about single quotes or double quotes for your strings. You have been mixing them and that won't cause an error, but you only want to pick one option and stick with it.